starter_code/views/layout.hbs

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="/stylesheets/styles.css">
+</head>
+
+<body>
+  <nav>
+
+    <div><a class="nav-link" href="/">Home</a>
+    </div>
+    <div> <a class="nav-link" href="/beers">Beers</a> </div>
+    <div> <a class="nav-link" href="/random-beers">Random Beer</a> </div>
+
+  </nav>
+
+  {{{body}}}
+
+
+  <footer>
+    <p class="footer-text">Develop by Ironhack ;)</p>
```

Comment:
again here, you could have targeted it just using the tag name `footer` rather than making a class